dietpi/dietpi-software

```diff
@@ -30,6 +30,29 @@ Available commands:
 	G_CHECK_ROOTFS_RW
 	G_INIT
 	# Import DietPi-Globals ---------------------------------------------------------------
+  
+  # Configuration Creation
+CONFIG_CREATE(){
+  [[ $G_PROGRAM_NAME ]] || local G_PROGRAM_NAME='G_CONFIG_CREATE'
+  local file=$1
+  local service=$2
+  local timeout=15 
```

Comment:
That timeout should be optionally configurable, since different services might take different startup times, e.g. via optional `$3`?